### ISSUE TITLE
Incompatible with AxisIndices v0.4

### ISSUE THREAD

**Issue by grahamas** [JuliaData/JuliaDB.jl#350]
Sorry, this is probably pretty niche:

I can't actually update to current master (v0.13.1) due to a compat conflict. I need AxisIndices v0.4, but a new compat requirement in master conflicts, below.  It looks like the conflicting requirement in this package is `IndexedTables = "= 0.12.4"`, which necessitates `Tables < v1`.  `IndexedTables@0.12.6` would resolve the conflict.

```
pkg> dev JuliaDB
  Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Tables [bd369af6]:
 Tables [bd369af6] log:
 ├─possible versions are: [0.1.0-0.1.15, 0.1.17-0.1.19, 0.2.0-0.2.11, 1.0.0-1.0.5] or uninstalled
 ├─restricted by compatibility requirements with AxisIndices [f52c9ee2] to versions: 1.0.0-1.0.5
 │ └─AxisIndices [f52c9ee2] log:
 │   ├─possible versions are: [0.1.0-0.1.4, 0.2.0-0.2.4, 0.3.0-0.3.12, 0.4.0-0.4.7, 0.5.0-0.5.1, 0.6.0-0.6.3] or uninstalled
 │   ├─restricted to versions * by TravelingWaveSimulations [b7a787de], leaving only versions [0.1.0-0.1.4, 0.2.0-0.2.4, 0.3.0-0.3.12, 0.4.0-0.4.7, 0.5.0-0.5.1, 0.6.0-0.6.3]
 │   │ └─TravelingWaveSimulations [b7a787de] log:
 │   │   ├─possible versions are: 2.0.0 or uninstalled
 │   │   └─TravelingWaveSimulations [b7a787de] is fixed to version 2.0.0
 │   └─restricted to versions 0.4.7 by an explicit requirement, leaving only versions 0.4.7
 └─restricted by compatibility requirements with IndexedTables [6deec6e2] to versions: 0.2.4-0.2.11 — no versions left
   └─IndexedTables [6deec6e2] log:
     ├─possible versions are: [0.7.4-0.7.5, 0.8.0-0.8.1, 0.9.0-0.9.1, 0.10.0, 0.11.0, 0.12.0-0.12.6, 0.13.0] or uninstalled
     └─restricted to versions 0.12.4 by JuliaDB [a93385a2], leaving only versions 0.12.4
       └─JuliaDB [a93385a2] log:
         ├─possible versions are: 0.13.1 or uninstalled
         ├─restricted to versions * by TravelingWaveSimulations [b7a787de], leaving only versions 0.13.1
         │ └─TravelingWaveSimulations [b7a787de] log: see above
         └─JuliaDB [a93385a2] is fixed to version 0.13.1
```

**Comment by jpsamaroo** [comment]
I intentionally set the upper bounds to exact versions in #347 to fix failing CI, so we'll probably just need a few iterations of loosening upper bounds to get into a much better place :smile:

**Comment by grahamas** [comment]
Fixed by #387  (thanks much; and apologies to @jpsamaroo that I never noticed the PR you tagged me in until it was closed this morning!)
